diff --git a/node_modules/react-native/ReactCommon/jsi/React-jsi.podspec b/node_modules/react-native/ReactCommon/jsi/React-jsi.podspec
index fb3cbe9..950479b 100644
--- a/node_modules/react-native/ReactCommon/jsi/React-jsi.podspec
+++ b/node_modules/react-native/ReactCommon/jsi/React-jsi.podspec
@@ -63,4 +63,8 @@ Pod::Spec.new do |s|
     s.dependency "hermes-engine"
   end
   s.exclude_files = files_to_exclude
+
+  s.public_header_files = [
+    "jsi/*.{h,hpp}"
+  ]
 end
